_sass/3-layout/_email.scss

```diff
@@ -0,0 +1,11 @@
+.emailPageHeader,
+.emailContentSection>.container {
+  @include siteWidth;
+}
+
+.emailContentSection {
```

Comment:
This could probably use some work in regards to vertical padding. The top padding for the email content seems to be significantly smaller than what's on the site right now, and the bottom padding seems to be zero. The page header / footers should match this padding